### ISSUE TITLE
leap: R, S, T unclear

### ISSUE THREAD

**Issue by btracey** [hybridgroup/gobot#187]
I started playing around with the leap motion API. It took me a long time to find what R, S, and T mean. They aren't described in the leap motion "Hands" API, only in the API overview. I think at least a comment saying "Rotation", "Scale" and "Translation" is in order. Better yet would be to rename, though I understand if you want to match the json.

**Comment by zankich** [comment]
@btracey I agree that some of the labels aren't exactly clear, but I do want to match what the leap motion responds with so that it's easier for people to follow the leap motion documentation. The leap motion platform in Gobot could for sure use more documentation to help with the obscurity of some of the labels though.
